app/build.gradle

```diff
@@ -7,6 +7,7 @@ apply from: './jacoco.gradle'
 android {
     lintOptions {
         warning 'InvalidPackage'
+        abortOnError false
```

Comment:
>Change the library version

not the library version it is latest one. Will have to change the android palette version in the library. I am not actually sure about how to do it.